doc/scripts/create_anchors

```diff
@@ -0,0 +1,32 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2013 - 2018 by the deal.II authors
+##
+## The original file is from the deal.II library. It has been modified for
+## Ginkgo.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE.md at
+## the top level directory of deal.II.
```

Comment:
We don't have to update it, as we copied it with that version, I think. And the purpose is not to keep it up to date with their recent changes, we will actually maybe change parts of these scripts with Pratik to try to have examples without the `example-<number>` naming scheme, so this will diverge a bit.